src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_signverify.c

```diff
@@ -235,59 +92,190 @@ static int32_t ConfigureSignVerifyTransform(SecTransformRef xform,
         return 0;
     }
 
-    if (includeHashAlgorithm)
-    {
-        CFStringRef cfHashName = NULL;
-        int32_t hashSize = 0;
+    return 1;
+}
+#endif
 
+// Legacy algorithm identifiers
+extern const SecKeyAlgorithm kSecKeyAlgorithmRSASignatureDigestPKCS1v15MD5; // = CFSTR("algid:sign:RSA:digest-PKCS1v15:MD5");
+
+static CFStringRef GetSignatureAlgorithmIdentifier(PAL_HashAlgorithm hashAlgorithm, PAL_SignatureAlgorithm signatureAlgorithm)
+{
+    if (signatureAlgorithm == PAL_SignatureAlgorithm_EC)
+    {
+        // ECDSA signatures are always based on digests. The managed implementation
+        // will always pre-hash data before getting here.
+        return kSecKeyAlgorithmECDSASignatureDigestX962;
+    }
+    if (signatureAlgorithm == PAL_SignatureAlgorithm_RSA_Pkcs1)
```

Comment:
Please leave a blank line between dissociated control flow blocks.

```suggestion

    if (signatureAlgorithm == PAL_SignatureAlgorithm_RSA_Pkcs1)
```